### PR TITLE
Switch to Graphs.jl instead of LightGraphs.jl

### LINKED ISSUE #45: Switch to Graphs.jl instead of LightGraphs.jl
Since LightGraphs.jl [seized development](https://github.com/sbromberger/LightGraphs.jl), which is now being continued at [Graphs.jl](https://github.com/JuliaGraphs/Graphs.jl), it might be better to make TikzGraphs.jl depend on Graphs.jl, in order to use Graphs.jl along with TikzGraphs.jl in other projects.

### PR DESCRIPTION
**Pull request by simsurace**
Closes #45

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -3,13 +3,13 @@ uuid = "b4f28e30-c73f-5eaf-a395-8a9db949a742"
 version = "1.2.0"
```
Comment (rejuvyesh) [inline Project.toml:3]: Will make sense to move the version to "1.3.0"?

Comment (mykelk) [reply Project.toml:3]: Yes

Comment (rejuvyesh) [inline Project.toml:3]:
```suggestion
version = "1.3.0"
```

**Comment by tawheeler** [comment]
Could we generate the figures in the ipynb? That way readers can see the output.

**Comment by tawheeler** [comment]
I updated the ipynb in #47. Sorry, I don't think I had permission to push to your branch.

**Pull request merged**